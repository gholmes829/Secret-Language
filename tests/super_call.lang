# inheritance

class Animal {
    void id() {
        print('I am an animal')
    }
}

class Wolf(Animal) {
    void id() {
        super.id()
        print('...I am also a wolf')
    }
}

num main() {
    let wolf = Wolf()
    wolf.id()

    return 0
}