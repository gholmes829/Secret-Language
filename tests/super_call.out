I am an animal
...I am also a wolf
